test/edu/csus/ecs/pc2/core/report/ReportsTest.java

```diff
@@ -278,6 +278,7 @@ public void testReportTitles(){
                 "Problem Group Assignment", //
                 "Scoreboard JSON Report", //
                 "Compare Primary with model Report", //
+                "Awards Report", //
```

Comment:
Why is this file being changed in this PR? The change in the file doesn't appear to have anything to do with the purpose of the PR...